### PR TITLE
feat: add light button

### PR DESCRIPTION
**Pull request by nookismile**

### CODE REVIEW

File: src/06_shared/ui/button/lightButton/lightButton.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<button class="light-button" type="button">
```
Comment (MariiaBel) [inline src/06_shared/ui/button/lightButton/lightButton.html:1]: Do not need 'lightButton' folder. All buttons will be in 'button' folder

Comment (MariiaBel) [reply src/06_shared/ui/button/lightButton/lightButton.html:1]: We need more styles for light-button.  How will it look with link and input ?
```
<a class='light-button'></a> 
<input type="button" class='light-button'>

File: src/06_shared/ui/button/lightButton/lightButton.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
+.light-button {
+  display: inline-block;
+  padding: 4px 8px;
+  font-size: var(--fs-base);
+  line-height: var(--lh-base);
+  color: rgb(var(--clr-pale-grey));
+  text-align: center;
+  background: transparent;
```
Comment (MariiaBel) [inline src/06_shared/ui/button/lightButton/lightButton.scss:8]: Why background  does not have colour? Buttons aspirants should be the same on different place.

File: src/stories/lightButton.css [hunk 3]
```diff
@@ -0,0 +1,20 @@
+.light-button {
```
Comment (MariiaBel) [inline src/stories/lightButton.css:1]: Need to delete this additional file. it is copy of /06_shared/ui/button/lightButton/lightButton.scss

File: src/stories/lightButton.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import './lightButton.css';
```
Comment (MariiaBel) [inline src/stories/lightButton.js:1]: change import on/06_shared/ui/button/lightButton/lightButton.scss

File: src/stories/lightButton.stories.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import { createLightButton } from './lightButton';
+
+export default {
+    title: "Component/lightButton",
```
Comment (MariiaBel) [inline src/stories/lightButton.stories.js:4]: Change it on 
```
title: "Component/Button",
```

All button will be in  Component/Button section

File: storybook-static/assets/Button-BfyGbg8N.css [hunk 6]
```diff
@@ -0,0 +1 @@
+.storybook-button{font-family:Nunito Sans,Helvetica Neue,Helvetica,Arial,sans-serif;font-weight:700;border:0;border-radius:3em;cursor:pointer;display:inline-block;line-height:1}.storybook-button--primary{color:#fff;background-color:#1ea7fd}.storybook-button--secondary{color:#333;background-color:transparent;box-shadow:#00000026 0 0 0 1px inset}.storybook-button--small{font-size:12px;padding:10px 16px}.storybook-button--medium{font-size:14px;padding:11px 20px}.storybook-button--large{font-size:16px;padding:12px 24px}
```
Comment (MariiaBel) [inline storybook-static/assets/Button-BfyGbg8N.css:1]: Need to clean assets folder. Do you need files from assets folder?

**Review by MariiaBel — COMMENTED** [review]
Check storybook-static folder/ Do you need all that files?

File: src/06_shared/ui/button/lightButton/lightButton.scss (hunk 2 above)
Comment (MariiaBel) [reply src/06_shared/ui/button/lightButton/lightButton.scss:8]: What will be with a button if I add more text? Maybe it is better to add max-width style

Comment (nookismile) [reply src/06_shared/ui/button/lightButton/lightButton.scss:8]: Button does not have background color in Figma
image
image

Comment (MariiaBel) [reply src/06_shared/ui/button/lightButton/lightButton.scss]: Ok, it is fine/ I was wrong

File: storybook-static/assets/Button-BfyGbg8N.css (hunk 6 above)
Comment (MariiaBel) [reply storybook-static/assets/Button-BfyGbg8N.css]: Y need to delete storybook-static folder

**Review by MariiaBel — APPROVED** [review]

**Pull request merged**